### PR TITLE
refactor and prepare for dockerize

### PR DESCRIPTION
**Pull request by zxp86021**
@passerbyid please give a look

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,39 +1,45 @@
...
+const webduino = require('webduino-js');
+const firebaseAdmin = require('firebase-admin');
+const axios = require('axios');
+const config = require('./env.js');
+
+const firebaseDatabaseURL = config.firebaseDatabaseURL;
+const token = config.token;
+const cameraURL = config.cameraURL;
+const groupChatId = config.telegramGroupChatId;
+const devGroupChatId = config.telegramDevGroupChatId;
```
Comment (x3388638) [inline index.js]:
```suggestion
const {
  firebaseDatabaseURL,
  token,
  cameraURL,
  telegramGroupChatId: groupChatId,
  telegramDevGroupChatId: devGroupChatId
} = config
```

File: index.js [hunk 2]
```diff
@@ -1,39 +1,45 @@
...
+
 app.use(bodyParser.urlencoded({
   extended: true
 }));
 app.use(bodyParser.json());
+app.get('/', (req, res) => res.json({ Status: status }));
+
+const server = app.listen(process.env.PORT || '4000', function () {
+  const host = server.address().address;
+  const port = server.address().port;
```
Comment (x3388638) [inline index.js]: unused?

File: index.js [hunk 3]
```diff
@@ -1,39 +1,45 @@
...
+});
+
+firebaseAdmin.initializeApp({
+  credential: firebaseAdmin.credential.cert(config.firebaseServiceAccountKeyFilePath),
```
Comment (x3388638) [inline index.js]: `firebaseServiceAccountKeyFilePath` can be also destructured at line 8

**Comment by passerbyid** [comment]
Should update README.

**Review by YuerLee — CHANGES_REQUESTED** [review]
Should update README.

**Comment by zxp86021** [comment]
@passerbyid @x3388638 plz give more look (?

**Review by x3388638 — COMMENTED** [review]
LGTM

**Review by YuerLee — APPROVED** [review]
讚讚

**Pull request merged**